mongomock/helpers.py

```diff
@@ -8,10 +8,34 @@
 
 try:
     from bson import (ObjectId, RE_TYPE)
+    from bson.tz_util import utc
```

Comment:
This is an anti-pattern. You don't know which of these statements failed with `ImportError`, and the consequences can be unexpected

Reply:
fixed ^^